lib/stack_master/aws_driver/cloud_formation.rb

```diff
@@ -11,7 +11,9 @@ def delete_stack(options)
       end
 
       def describe_stacks(options)
-        cf.describe_stacks(options)
+        retry_with_backoff do
+          cf.describe_stacks(options)
+        end
       end
```

Comment:
So that's to apply any dirty stack? Perhaps that could be a new feature/cmd line option.

```
stack_master -f apply --changed
stack_master -f diff --changed
```

I guess that would require two separate bits of work. One to allow `stack_master apply` to work without parameters and simply loop through stacks applying them. Then adding `--changed` to only operate on stacks if the stack is different to what is on cloudformation.

`-f` will be introduced in https://github.com/envato/stack_master/pull/71

Working out dependencies would require either manual declaration of dependencies in stack_master.yml or working it out by looking at parameter files and stack_output usage. That would be a welcome addition though :)